taxi/tests/test_admin.py

```diff
@@ -0,0 +1,49 @@
+from django.contrib.auth import get_user_model
+from django.test import TestCase, Client
+from django.urls import reverse
+
+
+class AdminTests(TestCase):
+    def setUp(self):
+        self.client = Client()
+        self.adin_user = get_user_model().objects.create_superuser(

```

Comment:
There's a typo in the variable name `adin_user`. It should be `admin_user` for clarity and correctness.